src/discretization/schemes/WENO/WENO.jl

```diff
@@ -0,0 +1,89 @@
+
+"""
+Implements the WENO scheme of Jiang and Shu.
+
+Specified in https://repository.library.brown.edu/studio/item/bdr:297524/PDF/ (Page 8-9)
+
+Implementation *heavily* inspired by https://github.com/ranocha/HyperbolicDiffEq.jl/blob/84c2d882e0c8956457c7d662bf7f18e3c27cfa3d/src/finite_volumes/weno_jiang_shu.jl by H. Ranocha.
+"""
+@inline function weno(II::CartesianIndex, s::DiscreteSpace, wenoscheme::WENOScheme, b, jx, u, dx::Number)
```

Comment:
why inline?

Reply:
Just a habit I established from earlier schemes, should it not?